File: aspnet-core/src/Timesheet.Application/APIs/Retros/Dto/RetroDto.cs
using Abp.AutoMapper;
using Abp.Domain.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;
using Timesheet.Anotations;
using static Ncc.Entities.Enum.StatusEnum;

namespace Timesheet.APIs.Retros.Dto
{
    public class RetroDto : Entity<long>
    {
        [ApplySearch]
        public string Name { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public DateTime Deadline { get; set; }
        public RetroStatus Status { get; set; }
    }
}

File: angular/src/app/service/api/model/retro-dto.ts
import { ActionDialog } from "@shared/AppEnums";
import { EnumSort } from "./retro-detail-dto";

export interface RetroDto {
  id: number;
  name: string;
  startDate: string;
  endDate: string;
  deadline: string;
  status: number;
}

export interface RetroActionDialogData {
  item: RetroDto;
  action: ActionDialog;
}

export enum EnumSortName {
  StartDate = "startDate",
}

export interface RetroSort {
  name: EnumSortName;
  value: EnumSort;
}
